docs/team/chrystalquek.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: page
+title: Chrystal Quek's Project Portfolio Page
+---
+
+## Project: Productiv
+
+Productiv is a desktop application for product managers to organise their product-related information. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+
+Given below are my contributions to the project.
+
+* **New Feature**: Added the ability to switch modes.
+ * What it does: allows the user to `switch` to the different modes of the app - `db` (dashboard), `dv` (deliverable), `m` (meeting) or `c` (contact). Subsequent commands are executed with respect to the current mode.
+ * Justification: This feature improves the user experience significantly because the user can just choose to see only deliverable, meeting or contact related information. This makes the application less cluttered and more organised. 
+ * Also the user can remember less commands, e.g. the `add` command word can be used to add a deliverable, meeting or contact, depending on the mode the user is currently in.
+ * Highlights: This feature was very difficult to implement and required significant restructuring of the entire application. I had to make the UI change whenever there was a switch in mode and ensure that the subsequent commands were passed to the correct `LogicManagers` for execution.
+ * As the command is very short, e.g. `switch db` instead of `switch dashboard`, it is customised to the needs of fast typists. Also, a warning message is shown when users attempt to switch to the same mode. These change were made after considering the recommendations from PED.
+ * Add screenshot?
+
+* **New Feature**: Update person to contact that is more useful for product-management.
+ * What it does: allows the user to keep track of the developers and stakeholders that are involved in the development of the product. Contacts can have a role - developer or stakeholder and also a description.
```

Comment:
Punctuation issue. May consider either:
-Contacts can have a role—developer or stakeholder—and also a description
-Contacts can have a role (developer or stakeholder) and also a description